### PR TITLE
Instruction function name mapping for Python Customized deviceShifu

### PR DESCRIPTION
**Pull request by Twpeak**
**What this PR does / why we need it**:

**Will this PR make the community happier**? 
Solved Unable to use customize deviceshifu #338
**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**How is this PR tested**
- [ ] unit test
- [ ] e2e test
- [ ] other (please specify)

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: pkg/deviceshifu/deviceshifuhttp/deviceshifuhttp.go [hunk 1]
```diff
@@ -232,12 +232,12 @@ func (handler DeviceCommandHandlerHTTP) commandHandleFunc() http.HandlerFunc {
 			}
 
 			rawRespBodyString := string(respBody)
-			_, shouldUsePythonCustomProcessing := deviceshifubase.CustomInstructionsPython[handlerInstruction]
+			InstructionFuncName, shouldUsePythonCustomProcessing := deviceshifubase.CustomInstructionsPython[handlerInstruction]
```
Comment (tomqin93) [inline pkg/deviceshifu/deviceshifuhttp/deviceshifuhttp.go:235]: let's not use capitalized `I` for variable names that are not being exported, same for other occurrences.

**Review by tomqin93 — CHANGES_REQUESTED** [review]
@Twpeak added 1 comment, please take a look
Also, please update the how-to guide for customized deviceShifu

**Review by tomqin93 — DISMISSED** [review]

**Review by kris21he — COMMENTED** [review]
LGTM

**Comment by kris21he** [comment]
@Twpeak this PR has too much commits and the change seems not correct, can we close this and init another one that have the clean changes?

**Review by kris21he — APPROVED** [review]

**Pull request merged**